test/influence_diagram.jl

```diff
@@ -163,9 +165,9 @@ Y_V["b", "a"] = 5
 Y_V["b", "b"] = 6
 @test Y_V  == [1 2 3; 5 6 4]
 add_utilities!(diagram, "V", Y_V)
-@test diagram.Y == [[1 2 3; 5 6 4]]
-@test_throws DomainError add_utilities!(diagram, "V", Y_V)
 
+@test string(values(diagram.Y)) == "Utilities[[1.0 2.0 3.0; 5.0 6.0 4.0]]"
```

Comment:
Could this comparison be cleaner using the data-field of the Utilities struct? I'm not a fan of making these strings for comparison's sake. `[u.data for u in values(diagram.Y)]` might work?